rooms/default/usual/orc_shop.py

```diff
@@ -23,11 +23,15 @@ def get_actions(user):
 	return [ 'Зуб', 'Палка', 'Парашок', 'Уйти' ]
 
 def action(user, reply, text):
+	teeths = user.get_item_by_name('tooth')
+	if teeths is None:
+		user_teeth_cnt = 0
```

Comment:
ternary operator will be better therer

Reply:
That's impossible here. `teeths` variable used two times.